src/Fhir.Anonymizer.R4.CommandLineTool/Microsoft.Health.Fhir.Anonymizer.R4.CommandLineTool.csproj

```diff
@@ -11,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Fhir.Anonymizer.R4.Core\Fhir.Anonymizer.R4.Core.csproj" />
+    <ProjectReference Include="..\Fhir.Anonymizer.R4.Core\Microsoft.Health.Fhir.Anonymizer.R4.Core.csproj" />
```

Comment:
Shall we change the directory name to project name?

Reply:
+1

Reply:
Thanks, updated.